### PR TITLE
a new design

### PR DESCRIPTION
**Pull request by ngdream**
a new design

### CODE REVIEW

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Hey there! This is off to a very good start!

A few notes:

image

We should stick with a purple background, IMHO.

image

Whatever background we use should stretch, not tile.

image

This section should have the same width.

**Comment by ngdream** [comment]
hello @naomi-lgbt  i did some improvement

File: index.html [hunk 1]
```diff
@@ -74,7 +74,12 @@
     </div>
 
     <header id="welcome">
-      <h1>Welcome to Carrigan's Community! 🔥</h1>
+      <div style="text-align: center; margin-top: 10px;">
+              <h1>Welcome to Carrigan's Community! 🔥</h1>
+      <p>  <a href="" class="button">join community</a></p>
```
Comment (naomi-lgbt) [inline index.html:79]:
```suggestion
      <p>  <a href="https://github.com/naomis-novas/community/issues/new?assignees=naomi-lgbt&labels=%E2%9C%89%EF%B8%8F+misc%3A+invite+to+org&template=join_org.yml&title=%5BORG%5D+Invite+me+to+the+organization" class="button">join community</a></p>
```

Here's the invite link.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by ngdream** [comment]
cool

**Pull request merged**